infra/site-cookbooks/cassandra-manual/templates/default/cassandra.yaml.rb

```diff
@@ -152,17 +152,6 @@
 # Defaults to the same value as permissions_validity_in_ms.
 # permissions_update_interval_in_ms: 2000
 
```

Comment:
I went back down to a cassandra 3.0.* yaml as it is the most "stable"

Reply:
Okey 👍